src/resources/elements/tokenDistribution/tokenDistribution.html

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section if.bind='data' class="tokenDistribution">
+    <header class="tokenDistributionHeading">
+      <h3 class="heading heading3">Token Distribution</h3>
```

Comment:
Our code guidelines ask not to use stuff like `header` and `h3`.  Please use div and the closest appropriate style from styles.scss. 

But as long as `header` is known not to introduce formatting, then I wouldn't object to it.